### DISCUSSION TITLE
Transforming from epsg:4326 to epsg:31468 produces a 2 meter offset according to QGIS

### DISCUSSION THREAD

**Discussion by arminus** [pyproj4/pyproj#814] [Q&A]
I have an input polygon in epsg:31468 which I need to transform to epsg:4326, the code I'm currently using is this:
```
project = pyproj.Transformer.from_proj(
    pyproj.Proj(init='epsg:31468')
    pyproj.Proj(init='epsg:4326'))
poly = transform(project.transform, poly)
```
Loading both polygons into QGIS, I get an offset of ~ 2 meters for every point, though.

I got a suggestion on [reddit /r/gis](https://www.reddit.com/r/gis/comments/mh1q6o/reprojection_offset_issues_between_epsg31468_and/), that I may also need a datum transformation by using a towgs parameter - but I don't know which value I need to pick.

And there was another comment wether pyproj actually allows datum transformations?

Any help is appreciated!

**Comment by snowman2** [comment]
`pyproj` does take into account datum transformations. The grids may help with accuracy: https://pyproj4.github.io/pyproj/stable/transformation_grids.html

> I may also need a datum transformation by using a towgs parameter

This is no longer needed in PROJ 6+ from what I understand. I would recommend asking the PROJ mailing list for clarification on that: https://lists.osgeo.org/mailman/listinfo/proj/

Side note: This is the recommended pyproj 2+ method for creating transformer to use with shapely/pygeos transform:
```python
project = pyproj.Transformer.from_crs("EPSG:31468", "EPSG:4326", always_xy=True)
poly = transform(project.transform, poly)
```

**Reply by snowman2** [reply]
https://gdalbarn.com/ : see section on "WGS84 Pivot"

**Reply by snowman2** [reply]
Forwarded: https://lists.osgeo.org/pipermail/proj/2021-April/010169.html
